Help this weird configure script find SDK files. The placeholder will be
replaced with the SDK path by the Portfile.

Fix:
checking whether does this OS have IPv6 stack... configure: error: Failed to detect IPv6 stack
--- configure.ac.orig	2022-01-17 22:29:25.000000000 -0600
+++ configure.ac	2022-01-17 22:54:48.000000000 -0600
@@ -1166,7 +1166,7 @@
 dnl ---------
 dnl KAME IPv6
 dnl ---------
-  if grep WIDE /usr/include/netinet6/in6.h >/dev/null 2>&1; then
+  if grep WIDE @SDKROOT@/usr/include/netinet6/in6.h >/dev/null 2>&1; then
     AC_DEFINE(KAME,1,KAME IPv6)
     AC_MSG_RESULT(KAME)
 dnl ------------------------------------
